### DISCUSSION TITLE
Welcome to Spankbang-Downloader Discussions!

### DISCUSSION THREAD

**Discussion by YisusChrist** [YisusChrist/spankbang-dl#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by watchkhawen** [comment]
hi i'm getting this when trying to pip install

ERROR: Could not find a version that satisfies the requirement spankbang-dl (from versions: none)
ERROR: No matching distribution found for spankbang-dl

**Reply by YisusChrist** [reply]
Hello @watchkhawen 

First of all, thank you for your interest in this project. Actually, Spankbang-DL was never published to PyPi, so you won't be able to install it using that method. You can even test this yourself if you [try to search it](https://pypi.org/search/?q=spankbang-dl) in the PyPi's public packages list. I still have to publish it and this will be available for everybody. I will let you know when it is done.

Meanwhile, I advise you to follow the steps described for [manual installation](https://github.com/YisusChrist/spankbang-dl?tab=readme-ov-file#manual-installation). If you are already using `pip`/`pipx`, the fastest is:

```sh
pipx install git+https://github.com/YisusChrist/spankbang-dl.git
```

Please confirm if this works on your side.
